.github/workflows/release.yml

```diff
@@ -34,14 +34,14 @@ jobs:
       -
         name: Import GPG key
         id: import_gpg
-        uses: hashicorp/ghaction-import-gpg@v2.1.0
+        uses: crazy-max/ghaction-import-gpg@v5.0.0
```

Comment:
This is changed as https://github.com/hashicorp/ghaction-import-gpg is deprecated and as per the instructions used crazy-max/ghaction-import-gpg as advised.